source/braille.py

```diff
@@ -373,15 +374,22 @@
 
 
 def NVDAObjectHasUsefulText(obj: "NVDAObject") -> bool:
+	"""Does obj contain useful text to display in braille
+
+	:param obj: object to check
+	:type obj: NVDAObject
+	:return: True if there is useful text, False if not
+	:rtype: bool

```

Comment:
[Providing type information in docstrings is discouraged, instead use python's type annotations.](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#docstrings)

```suggestion
	:param obj: object to check
	:return: True if there is useful text, False if not
```